docs/source/zh/consisid.md

```diff
@@ -0,0 +1,100 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+# ConsisID
```

Comment:
Wow really nice of you to also add a translated version of the docs! 🤩

Reply:
thanks a lot, I have changed the `consisid_zh` to be the same formalized as `consisid_en`, e.g., _video_, _paper_ and _cite_.